### PR TITLE
updated location of recovery document

### PR DESCRIPTION
**Pull request by arindamnayak**

### CODE REVIEW

**Comment by deepthi** [comment]
One more request. Can we list MySQL & ripple under pre-conditions?
Or right at the top we can put a "Supported databases" and list them there. It should be very clear to people that MariaDB is not supported, nor have we tested with any binlog server other than ripple.

**Comment by arindamnayak** [comment]
> One more request. Can we list MySQL & ripple under pre-conditions?
> Or right at the top we can put a "Supported databases" and list them there. It should be very clear to people that MariaDB is not supported, nor have we tested with any binlog server other than ripple.

@deepthi , updated the document, please have a look. I havn't put the mysql and ripple together, Mentioned mysql at top and put ripple as pre-condition saying `currently it is tested with ripple` .

File: content/en/docs/reference/features/recovery.md [hunk 1]
```diff
@@ -5,6 +5,9 @@ aliases: ['/docs/recovery/pitr','/docs/reference/pitr/']
 
 ## Point in time recovery
 
+### Support Databases
```
Comment (deepthi) [inline content/en/docs/reference/features/recovery.md]:
```suggestion
### Supported Databases
```

**Review by deepthi — CHANGES_REQUESTED** [review]

File: content/en/docs/reference/features/recovery.md (hunk 1 above)
Comment (arindamnayak) [reply content/en/docs/reference/features/recovery.md]: done

**Review by deepthi — APPROVED** [review]

**Pull request merged**